backend/infrastructure/vector_store/vector_store.py

```
@@ -251,6 +295,347 @@ def benchmark(self, queries: np.ndarray, k_values: List[int] = [1, 5, 10]) -> Di
+        return None
+
+    def get_all_chunks(self) -> List[DocumentChunk]:
+        """Return all chunks cached from Qdrant payloads."""
+        return list(self.chunks)
+
+    def close(self) -> None:
+        if self._closed:
+            return
+        try:
+            self.client.close()
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)